config/initializers/paperclip.rb

```diff
@@ -1,3 +1,8 @@
-# Paperclip.interpolations[:slug] = proc do |attachment, style|
-#   attachment.instance.slug
-# end
\ No newline at end of file
+require 'paperclip/media_type_spoof_detector'
+module Paperclip
+  class MediaTypeSpoofDetector
+    def spoofed?
+      false
+    end
+  end
+end
```

Comment:
All files should end with a newline: http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline